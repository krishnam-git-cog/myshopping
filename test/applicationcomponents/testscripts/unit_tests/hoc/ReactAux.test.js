import React from 'react';
import { createShallow } from '@material-ui/core/test-utils';
import ReactAux from '../../../../../src/hoc/ReactAux';


describe('ReactAux test', () => {

    it('ReactAux should match snapshot', () => {
        let shallow = createShallow();
        const component = shallow(<ReactAux />);
        expect(component).toMatchSnapshot();
    });
});