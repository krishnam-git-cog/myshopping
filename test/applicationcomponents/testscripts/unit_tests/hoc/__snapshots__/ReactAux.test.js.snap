// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ReactAux test ReactAux should match snapshot 1`] = `ShallowWrapper {}`;
